apps/microtvm/poetry.lock

```diff
@@ -1,28 +1,9 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-# This `poetry.lock` file is generated from `poetry install`.
-
 [[package]]
 name = "absl-py"
-version = "1.2.0"
+version = "1.3.0"
```

Comment:
How these changes in `apps/microtvm` are related to adding "Open with Colab" button?

Reply:
cc @mehrdadh.

Reply:
This was a hack, please revert it.

Reply:
Fixed.